src/main/webapp/app/shared/monaco-editor/model/themes/monaco-editor-theme.model.ts

```
@@ -0,0 +1,66 @@
+            'diffEditor.removedTextBackground': colorDefinitions.diff?.removedTextBackgroundColor,
+            'diffEditor.removedLineBackground': colorDefinitions.diff?.removedLineBackgroundColor,
+            'diffEditor.diagonalFill': colorDefinitions.diff?.diagonalFillColor,
+            'diffEditorGutter.insertedLineBackground': colorDefinitions.diff?.gutter?.insertedLineBackgroundColor,
+            'diffEditorGutter.removedLineBackground': colorDefinitions.diff?.gutter?.removedLineBackgroundColor,
+        };
+
+        const tokenStyleDefinitions = this.themeDefinition.tokenStyles;
+        const rules = tokenStyleDefinitions.map((tokenDefinition) => {
+            // Language-specific tokens have the key `token.languageId`, e.g. keyword.custom-md
+            return {
+                token: `${tokenDefinition.token}${tokenDefinition.languageId ? '.' + tokenDefinition.languageId : ''}`,
```

Biome

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)